tests/test_server.py

```diff
@@ -119,7 +119,7 @@ def get_notification(self):
 class EvilClient(SimpleClient):
     """An evil line receiving and sending client."""
 
-    def call(self, call):
+    def call(self, call):  # pylint: disable=arguments-differ
```

Comment:
god, wonder why im asking this. but why dont we put in the args and kwargs so that they dont differ?